### ISSUE TITLE
Is silc dead?

### ISSUE THREAD

**Issue by echto** [silc/silc#9]
Is silc no longer under development?

**Comment by deavmi** [comment]
I would like to know too.

**Comment by fuzzy76** [comment]
No code changes for over 4 years... I think that answers your question. [Telegram](https://telegram.org) group chats or an IRC-client that support [Off-The-Record encryption](https://otr.cypherpunks.ca) might solve your problems.

**Comment by deavmi** [comment]
Well I mean silc does work and if you choose correct branch there were changes in 2017 but idk if they were code

**Comment by fuzzy76** [comment]
They were only to the build scripts, not actual code. :-/

**Comment by deavmi** [comment]
Sad

**Comment by tabesin** [comment]
Maybe the software is just "finished".

**Comment by ptman** [comment]
SILC was nice, but seems to have faded away. Maybe matrix.org is a better modern alternative with activity

**Comment by ptman** [comment]
@priikone

**Comment by c0dev0id** [comment]
I thought of silc again, because Matrix is way too fat already (too many features). I remember when we had the CCC conference on silc with a few thousand participants. It was nice!

Even today, silc would be a nice fit between the new and bloated hipster networks and the good old IRC network.

**Comment by ptman** [comment]
@c0dev0id is matrix too fat? or the clients? there are slim clients as well

I too have a fond memory of silc, and hope that matrix will still adopt a few good ideas. Like having user identity be the keypair.

**Comment by deavmi** [comment]
Wait.... with video?

On 10 November 2020 10:33:40 SAST, sdk <notifications@github.com> wrote:
>I thought of silc again, because Matrix is way too fat already (too
>many features). I remember when we had the CCC conference on silc with
>a few thousand participants. It was nice!
>
>Even today, silc would be a nice fit between the new and bloated
>hipster networks and the good old IRC network.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/silc/silc/issues/9#issuecomment-724548540

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by c0dev0id** [comment]
> Wait.... with video?

No, streaming came later. Back then the conference was live with about 3k people. Videos came weeks later as downloads. But we used silc as the preferred conference + external chat.

**Comment by c0dev0id** [comment]
> @c0dev0id is matrix too fat? or the clients? there are slim clients as well

Both. The synapse server is a hell of a python beast. There's light at the end of the tunnel: dentrite, which is a reimplementation in go (it still needs 1000% of the resources of an IRC or Silc Server).

What really hurts is that the matrix protocol demands clients to implement a minimal set of features. This minimal set is quite fat and difficult to handle in terminals. Like double width utf8 characters everywhere, reply feature, editing existing texts, reactions, images etc. There is a terminal client that handles it reasonably well: gomuks, but it still craps out in a lot of terminals. Especially because of those double width UTF8 characters and people using it in their user names. If some of the features could be forbidden for some rooms, that would be wonderful. But Matrix devs are deaf on this ear. They want to push their idea to be the standard, and everyone else needs to adapt. (this is what makes the matrix-irc bridge so painful).

Anyway... silc was nice because it was IRC-like, just a bit better, instead of reinventing the wheel. And it was very console friendly (which is very important for me, because I can access chat services only via ssh most of the time).

**Comment by daiwei-china** [comment]
I need a realse for runing it as sever in my router，linux ARM64 （openwrt）aarch64.  IT IS TOO  DIFFICULT TOO BUILD IT ！

**Comment by deavmi** [comment]
I should look at this project again, I now am way smarter than when I made my original comment.
